packages/shared/src/components/profile/UserList.tsx

```diff
@@ -50,8 +53,9 @@ function UserList({
               tag="a"
               href={user.permalink}
               user={user}
+              endChildren={additionalContent?.(user, i)[1]}
             >
-              {additionalContent?.(user, i)}
+              {additionalContent?.(user, i)[0]}
```

Comment:
why 0 here?

Reply:
Index zero on the array of components 😅
That's why I mentioned it's not super readable/clean

Reply:
Yeah no, that's why I wasn't expecting this 😅 was expecting all of the return array to be mapped here somewhere